src/sentry/api/endpoints/organization_dynamic_sampling_sdk_versions.py

```diff
@@ -65,14 +66,18 @@ def get(self, request: Request, organization) -> Response:
         )
         start_time = end_time - stats_period
 
-        avg_equation = 'count_if(trace.client_sample_rate, notEquals, "") / count()'
+        sample_rate_count_if = 'count_if(trace.client_sample_rate, notEquals, "")'
+        avg_sample_rate_equation = f"{sample_rate_count_if} / count()"
+        transaction_source_count_if = 'count_if(transaction.source, equals, "unknown")'
```

Comment:
There are a bunch of valid sources, we're looking for all those where the source is not empty.

```suggestion
        transaction_source_count_if = 'count_if(transaction.source, enotEquals, "")'
```

The sources map to these cases:
- "": old SDK with probable high cardinality, should be upgraded
- unknown: old SDK with improbable high cardinality, OK
- url: new SDK with high cardinality, handled in ingestion, OK
- anything else: new SDK with low cardinality, OK